src/main/scala/net/cakesolutions/CakeDockerComposePlugin.scala

```diff
@@ -50,7 +50,11 @@ object CakeDockerComposePlugin extends AutoPlugin {
         yaml => Seq("-f", yaml.getCanonicalPath)
       )
     val result =
-      Process(Seq("docker-compose") ++ projectOverrides ++ Seq("down")).!
+      Process(
```

Comment:
We should also add in a `--force-recreate` option on the docker-compose up - that way we are sure that images are actually rebuilt and stale images in CI docker registry and not used. For multiple CI runs, we probably want this to be something we can switch on/off? Maybe we should add in an extra settings key for this?